pom.xml

```diff
@@ -10,15 +10,15 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.10.1</version>
+        <version>3.13.0</version>
```

Comment:
using jitpack ci, 3.21.1 is maximum version w/o special config.

Reply:
Ok, I wasn't aware of the Jitpack restrictions. I downgraded to 3.12.1